### ISSUE TITLE
Date recorded in Ontology OWL file

### ISSUE THREAD

**Issue by justgo129** [USGCRP/gcis-ontology#183]
Is there a way to invert the date that is present in the header of the [ontology file](https://data.globalchange.gov/gcis.owl)?  It reads 03/11/2015, when it's should be dated November 11, 2015.  The [line in the ttl file](https://github.com/USGCRP/gcis-ontology/blob/master/gcis.ttl) reads:

"dcterms:date "2015-11-03"^^xsd:date ;

**Comment by zednis** [comment]
I do not know, check the LODE documentation - http://www.essepuntato.it/lode

If there is no available configuration parameter to control this, we could always fork the code (on github at https://github.com/essepuntato/LODE) and create a version that generates the date with our preferred formatting.

**Comment by justgo129** [comment]
Good idea.  There's nothing in the LODE documentation.  I believe the parameter is located on line 281 at:
https://github.com/essepuntato/LODE/blob/4a3860b01aa8fc6fe80147de1816c00efee1aee3/src/main/webapp/extraction.xsl

 <xsl:when test="matches(.,'\d\d\d\d-\d\d-\d\d')">

**Comment by zednis** [comment]
If we are using static HTML generated by LODE we can manually update the HTML to fix the published webpage, then we can fork LODE and tweak it so it generates what we want in the future.  We may additionally be able to incorporate the anchor link fix into our modified copy of LODE.

**Comment by justgo129** [comment]
@rewolfe what do you think?
